ext/pcntl/pcntl.c

```diff
@@ -1067,6 +1067,20 @@ static void pcntl_siginfo_to_zval(int signo, siginfo_t *siginfo, zval *user_sigi
 				add_assoc_long_ex(user_siginfo, "fd",   sizeof("fd")-1,   siginfo->si_fd);
 # endif
 				break;
+#endif
+
+#ifdef SIGTRAP
+			case SIGTRAP:
+# if defined(si_syscall) && defined(__FreeBSD__)
+				if (siginfo->si_code == TRAP_CAP) {
```

Comment:
I don't have FreeBSD, and I have no idea what our lowest supported version is, but is this `TRAP_CAP` constant old enough that it won't cause build failures on older FreeBSD's that we support?

Reply:
AFAIK TRAP_CAP appeared before the si_syscall shortcut.